docs/faq.md

```diff
@@ -30,17 +12,6 @@ If it gives you an error, you don't have it installed. You can download it [here
 
 Paper 1.17 or above requires Java 11 ([info here](https://papermc.io/forums/t/java-11-mc-1-17-and-paper/5615)).
 
-<a name="server-keeps-saying-starting"></a>
-## My server keeps saying starting.
-
-If your server has actually started but mcss isn't detecting it then you can change the way how mcss detects online servers.
-
-Go to `File > Options > Advanced` and change the default detection method to `ConsoleReading`.
-
-![Screenshot of the process name requires different format dialog](assets/screenshots/online_detection.png)
-
-
-<a name="unable-to-connect-to-the-server"></a>
 ## Unable to connect to the server.
 
 As the server owner, try connecting with "localhost" (without the ""). 
```

Comment:
That works only if the server is hosted on the same machine as the client. That could not be the case.

Reply:
Sure, editing now.